core/src/main/tut/README.md

```diff
@@ -101,15 +101,46 @@ As a result we recommend only using the latest Scala versions within the minor s
 
 ## Use PureConfig
 
-Import the library package and use one of the `loadConfig` methods:
+First, import the library, define data types, and a case class to hold the configuration:
+
+```scala
+import pureconfig.loadConfig
+
+sealed trait MyAdt
+case class AdtA(a: String) extends MyAdt
+case class AdtB(b: Int) extends MyAdt
+final case class Port(value: Int) extends AnyVal
+case class MyClass(
+  boolean: Boolean,
+  port: Port,
+  adt: MyAdt,
+  list: List[Double],
+  map: Map[String, String],
+  option: Option[String])
+```
 
-```tut:silent
-import pureconfig._
-import pureconfig.error.ConfigReaderFailures
+Second, define a configuration. Options for defining this are described in
+the [config files documentation](docs/config-files.md):
 
-case class YourConfClass(name: String, quantity: Int)
+`application.json`
+```json
+{ 
+  "boolean": true,
+  "port": 8080, 
+  "adt": { 
+    "type": "adtb", 
+    "b": 1 
+  }, 
+  "list": ["1", "20%"], 
+  "map": { "key": "value" } 
+}
+```
+
+Then, load the configuration ([in this case from the classpath](docs/config-files.md)):
 
-val config: Either[pureconfig.error.ConfigReaderFailures,YourConfClass] = loadConfig[YourConfClass]
+```scala
```

Comment:
My only concern with this particular change is that it's not being run by tut anymore. While I understand that loading a config from a file is much more useful than from a string for showcase purposes, this will get outdated if we ever change the `loadConfig` signature or the behavior of one of the readers.

I'm willing to let this without tut as it's a very short code section and we shouldn't change such an entrypoint to our library anyway. @melrief, do you agree?

Reply:
Most of the people use pureconfig to load from `application.conf` and I think @skeet70 changes are good. I'd like to have `tut` to check also this example but we can look into this later. Maybe there is way to still use `tut` somehow. For now, let's keep it unchecked @ruippeixotog.